code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm

```diff
@@ -64,6 +64,15 @@
 	weed_food_states = list("Facehugger_1","Facehugger_2","Facehugger_3")
 	weed_food_states_flipped = list("Facehugger_1","Facehugger_2","Facehugger_3")
 
+/mob/living/carbon/xenomorph/facehugger/Initialize(mapload, mob/living/carbon/xenomorph/old_xeno, hivenumber)
```

Comment:
This doesn't handle the situation of freed mobs or taking over an AFK xeno.

You probably want to use /mob/living/carbon/xenomorph/lesser_drone/Login instead or have this xeno register to COMSIG_GLOB_MOB_LOGIN to add the component. Or just always have the component on this xeno type and have it register to COMSIG_GLOB_MOB_LOGIN to start its say blocking & wait timer.